### ISSUE TITLE
cross compiling on linux/ubuntu

### ISSUE THREAD

**Issue by zashraf1337** [spender-sandbox/cuckoomon-modified#25]
Greetings,

Has anyone successfully compiled it on linux/ubuntu?

I see some files including `Windows.h` instead of `windows.h` which does not work on Ubuntu.

Other errors are also emitted.

**Comment by doomedraven** [comment]
nop, i have vm for it

**Comment by KillerInstinct** [comment]
Yeah it's been a long time since gcc was 'supported'. Once the debug code was added, (exception handling crashes) it stopped working I believe. Best to use Visual Studio Community 2015, it's free with any MS account (outlook/hotmail/etc.).
